docs/admin/access.rst

```diff
@@ -128,7 +129,7 @@ to all users.
     For non-`Custom` access control an instance of each group described above is
     actulally defined for each project. The actual name of those groups will be
     ``Project@Group``, also displayed in the Django admin interface this way.
-    Although they can't be edited from there.
+    Although they can't be edited from Weblate user interface.
```

Comment:
user-interface

Reply:
I would keep *Weblate* here intentionally to stay clear as we refer to the Django admin user interface in the lines around.

Reply:
That isn't how we agreed to hyphen it in the rework 2 PR. I won't miss it in 2.5.

Reply:
I see, you’re right! I thought you meant removing the *Weblate* word

Reply:
done